src/lib/wallet/GoodWalletClass.js

```diff
@@ -594,9 +631,16 @@ export class GoodWallet {
    * @param {string} link
    * @returns {Promise<boolean>}
    */
-  isWithdrawLinkUsed(link: string): Promise<boolean> {
-    const { hasPayment } = this.oneTimePaymentsContract.methods
-    return hasPayment(link).call()
+  async isWithdrawLinkUsed(link: string): Promise<boolean> {
+    try {
+      const { hasPayment } = await this.oneTimePaymentsContract.methods
```

Comment:
unneded destructiring, just `return this.oneTimePaymentsContract.methods.hsPayment(link).call()`